Added more infos about installation on Debian to README (#2241)
xournalpp/xournalpp: README.md

With the help of the community, Xournal++ is also available on official repositories
of some popular Linux distros and platforms.

### Debian

> [Stable releases](https://github.com/xournalpp/xournalpp/releases) and
> _unstable_, [automated nightly releases](https://github.com/xournalpp/xournalpp/releases/tag/nightly)
> for Debian can be found on Debian.

Reviewer: I am not a native English speaker, but I think this ',' is not necessary.

```suggestion
There are [Stable releases](https://github.com/xournalpp/xournalpp/releases) and
_unstable_ [automated nightly releases](https://github.com/xournalpp/xournalpp/releases/tag/nightly)
for Debian.
```
